### ISSUE TITLE
Unable to install on CentOS 7

### ISSUE THREAD

**Issue by goginoida** [redis/RedisDesktopManager#3706]
RDM Version: 0.8.0 <br>
Environment (OS name and version): Linux CentOS 7 <br>
Redis Server Version: 2.8 <br>

Steps to reproduce:
1. Clone the source using: 
   `git clone --recursive https://github.com/uglide/RedisDesktopManager.git -b 0.8.0 rdm && cd ./rdm/src`
2. Run the configure script:
   `./configure`

Expected result: Configure script should successfully complete

Actual Result: Following error message is received immediately
`./configure: line 63: [[: 7.2.1511: syntax error: invalid arithmetic operator (error token is ".2.1511")
Your operation system is not supported!
`

CentOS is a supported OS hence it should not complain.
Output of uname -a
`Linux machine-hostname 3.10.0-327.28.2.el7.x86_64 #1 SMP Wed Aug 3 11:11:39 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux`

**Comment by davidxifeng** [comment]
I have the same issue.

and here is my fix:

TL;DR

``` bash
* editor `cofigure` script,  goto line 14, right after `GetOSVersion`,
  add a one line hack: `os_RELEASE=7`

then just follow the building guide:

./configure` && qmake-qt5 && make
```

explain:

os_RELEASE on CentOS 7 is a string like `7.2.1511`, but the configure script expect just a string like `6` or `7`, so this will failed:

``` bash
"$os_RELEASE" -ge "7"
```

instead of modify the configure script, you can just run deps of CentOS 7 yourself with the following packages in the configure script:

```
    sudo yum install libssh2 libssh2-devel qt5-qtbase qt5-qtsvg-devel \
        qt5-qtdeclarative-devel qt5-qtgraphicaleffects qt5-qtquickcontrols  \
        qt5-qttools gcc gcc-c++ libstdc++-static git cmake rsync \
        redhat-rpm-config -y
```

and the build and install third party deps yourself, too.

**Comment by davidxifeng** [comment]
a possible fix is just remove the os_RELEASE check, does it looks good for you @uglide ?

**Comment by uglide** [comment]
Fixed by @DavidFeng in 0.8.0
